security/integration/main.go

```diff
@@ -73,185 +73,182 @@ func init() {
 	flags.StringVar(&opts.containerImage, "image", "", "Name of Istio CA image")
 	flags.StringVar(&opts.containerTag, "tag", "", "Tag for Istio CA image")
 	flags.StringVarP(&opts.kubeconfig, "kube-config", "k", "~/.kube/config", "path to kubeconfig file")
-	flags.StringVar(&opts.org_root_cert, "root-cert", "", "Path to the original root ceritificate")
-	flags.StringVar(&opts.org_cert_chain, "cert-chain", "", "Path to the original certificate chain")
+	flags.StringVar(&opts.orgRootCert, "root-cert", "", "Path to the original root ceritificate")
+	flags.StringVar(&opts.orgCertChain, "cert-chain", "", "Path to the original workload certificate chain")
 
 	cmd.InitializeFlags(rootCmd)
 }
 
 func main() {
 	// HACKHACK: let `flag.Parsed()` return true to prevent glog from emitting errors
 	flag.CommandLine = flag.NewFlagSet("", flag.ContinueOnError)
-	if err := flag.CommandLine.Parse([]string{}); err != nil {
-		glog.Fatal(err)
+	if errors := flag.CommandLine.Parse([]string{}); errors != nil {
```

Comment:
What's reason for renaming `err` to `errors`? I believe `err` is more widely used.

Reply:
+1, errors is not accurate, it should be error, but since error can stand for a lib, usually people use err for golang.

Reply:
Thanks for the comments. I have changed errors back to err. 
Linter complains that at line 167 and line 173, declaration of "err" shadows declaration at security/integration/main.go:146 (vetshadow), so I rename these two err to delErr and creErr respectively.

Reply:
It's because you are trying to declare an already existed var. 

For declaration, you should use
err := ***

And after the err is declared,  you should use

err = *** (no ':' any more)